### ISSUE TITLE
Better error message on accidental import of wrong series

### ISSUE THREAD

**Issue by CAYdenberg** [uber/react-vis#1049]
I mistakenly imported BarSeries instead of VerticalBarSeries in my component file. Easy mistake to make.

Unfortunately the error message is 
`Cannot read property 'prototype' of undefined`

which occurs when react-vis is checking the children of XYPlot to see if they are a series or not. It would be cool if it could warn you you've done something terrible instead.

I'd be happy to contribute if you guys will take it.

**Comment by mcnuttandrew** [comment]
Good idea! I would be delighted to see a PR to this effect.

**Comment by Pardeep94** [comment]
May i know what is the problem with BarSeries?
I also use BarSeries and get same error.
when i am using VerticalBarSeries than it is working fine

**Comment by CAYdenberg** [comment]
Yeah, the error is that there **is** no component called `BarSeries` in react-vis ... there is however a `VerticalBarSeries` and a `HorizontalBarSeries`.

It's a somewhat easy mistake to make because of the examples in the documentation.

**Comment by Pardeep94** [comment]
Ok, Thank you
